File: ch1/py/b-chat.py
from langchain_anthropic import ChatAnthropic
from langchain_core.messages import HumanMessage

model = ChatAnthropic(model="claude-haiku-4-5")
prompt = [HumanMessage("What is the capital of France?")]

response = model.invoke(prompt)
print(response.content)

File: ch1/js/b-chat.js
import { ChatAnthropic } from '@langchain/anthropic';
import { HumanMessage } from '@langchain/core/messages';

const model = new ChatAnthropic({ model: 'claude-haiku-4-5' });
const prompt = [new HumanMessage('What is the capital of France?')];

const response = await model.invoke(prompt);
console.log(response);
